### PR TITLE
Add TIM icon

### PR DESCRIPTION
**Pull request by fransdhinta**
tim

**Issue:** #11735

**Popularity metric:**
Similarweb Global Rank: [1,147](https://www.similarweb.com/website/tim.it)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

- chose that color `#eb0028` because it is the most prominent color in the logo.

### CODE REVIEW

**Comment by fransdhinta** [comment]
hi @uncenter, i have adjusted the attribute order & centering the path

**Comment by LitoMore** [comment]
I will help you guys out.

**Comment by LitoMore** [comment]
I rebased and force force pushed to fix those conflicts for you.

I also fixed the missing title, and removed the unecessary `TIM` text part from the icon.

Here is the preview:

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15873,6 +15873,11 @@
             "hex": "000000",
             "source": "https://www.thetileapp.com"
         },
+        {
+            "title": "TIM",
+            "hex": "EB0028",
+            "source": "https://risorse.tim.it/content/dam/flytoco-areapubblica-aemfe/loghi/logo.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:15879]: As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines, this must be a link to the page containing the SVG - not directly to it.
```suggestion
            "source": "https://tim.it"
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
See https://github.com/simple-icons/simple-icons/issues/11735#issuecomment-2353695877. We will need to reach out and request permission from TIM (see https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission) before adding this. @fransdhinta would you be willing to send an email / contact them?

**Comment by fransdhinta** [comment]
> See [#11735 (comment)](https://github.com/simple-icons/simple-icons/issues/11735#issuecomment-2353695877). We will need to reach out and request permission from TIM (see https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission) before adding this. @fransdhinta would you be willing to send an email / contact them?

yes i will send permission request

**Comment by fransdhinta** [comment]
@uncenter i have sent the request but they reply like this:

> Messaggio rifiutato dal sistema
> La casella destinataria di seguito indicata riceve esclusivamente messaggi di posta elettronica certificata telecomitalia@pec.telecomitalia.it – casella che riceve solo Posta Certificata

**Comment by adamrusted** [comment]
@fransdhinta - if we don't have a way of getting in touch with the brand, I'm afraid we'll have to close this PR. Would you be willing to reach out to them on social media, if their mailboxes aren't accepting external messages? If so, we can add a bit more time to the clock before closing - but if we've exhausted options, then we'll have to close the PR.

**Comment by fransdhinta** [comment]
> @fransdhinta - if we don't have a way of getting in touch with the brand, I'm afraid we'll have to close this PR. Would you be willing to reach out to them on social media, if their mailboxes aren't accepting external messages? If so, we can add a bit more time to the clock before closing - but if we've exhausted options, then we'll have to close the PR.

Okay, I will try to contact them on social media.

**Comment by fransdhinta** [comment]
@adamrusted  - I also can't contact them on social media, I think you can close this PR

**Pull request closed without merging**